### DISCUSSION TITLE
Exclude PodVolumes from Backup

### DISCUSSION THREAD

**Discussion by muratyarali** [vmware-tanzu/velero#8149] [Community support Q&A]
Is the opt-out-approach the only way to exclude PodVolumes from backup? I need to exclude all PodVolumes not a specific one.

https://velero.io/docs/v1.13/file-system-backup/#using-the-opt-out-approach

**Comment by sseago** [comment]
If you're just trying to exclude from fs-backup (i.e. to back up via snapshots instead), which is what the opt-out approach does, then it sounds like you want the opt-in approach. Opt-out says "use fs-backup by default, except for opted-out volumes, which will use snapshots" -- the opt-in approach says "use snapshots by default, except for opted-in volumes, which will use fs-backup".

In other words, opt-in with no opted in volumes is equivalent to opt-out and opting out every volume.
